youtube_dl/extractor/ibm_think.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from .generic import GenericIE
+
+
+class IbmThinkIE(InfoExtractor):
+    IE_DESC = 'IBM Think Videos'
+    IE_NAME = 'IBMThink'
+    _VALID_URL = r'https?://(?:www\.)?ibm\.com/events/think/watch/(playlist/)?(\d*/)?replay/(?P<id>[0-9]+)/?'
+    _TESTS = [{
+        'url': 'https://www.ibm.com/events/think/watch/replay/113734399/',
+        'md5': '0a3f1c81c58aacbbb36e292a1c1f9690',
+        'info_dict': {
+            'id': '113734399',
+            'ext': 'mp4',
+            'title': 'Think 2018 Chairman\'s Address: Putting Smart to Work',
+            'timestamp': 1521575552,
+            'upload_date': '20180320',
+            'uploader': 'f8k4md3yana',
+            'uploader_id': '43178333',
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        ustream_url = self._html_search_regex(r'<iframe\ssrc="(.*?)"', webpage, 'ustream_url').split('?')[0] + '/'
+        return self.url_result(ustream_url, GenericIE.ie_key())
```

Comment:
Nothing changed.

Reply:
I am using the generic ie here as requested.

Reply:
This was address to complete extractor.

Reply:
I am unable to get the issue. Please explain in a little more detail as i am new to this project.

Reply:
Generic iframe embeds must be extracted by `GenericIE`. Do not add new extractors for this.